### PR TITLE
S01 us08 creation of the registration page

### PR DESCRIPTION
**Pull request by Thomas-Lunardo**

### CODE REVIEW

**Review by Merhene — APPROVED** [review]

File: src/View/Home/login.html.twig [hunk 1]
```diff
@@ -1,5 +1,23 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-    <h1>page de connexion</h1>
-{% endblock %}
+	<div class="container loginPage">
```
Comment (benrcd) [inline src/View/Home/login.html.twig]: Unclosed div

File: src/View/Home/login.html.twig [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+			<form action="" method="post">
+				<label for="email">Votre email :
+				</label>
+				<input type="email" id="email" name="email" required placeholder="lamarquise@gmail.com">
+				<fieldset>
+					<label for="password">Votre mot de passe :
+					</label>
+					<input type="password" id="password" name="password" required>
+				</fieldset>
+				<div class="buttonsLine">
```
Comment (benrcd) [inline src/View/Home/login.html.twig]: HTML classes should be written this way "buttons-line"

File: src/View/Home/signup.html.twig [hunk 3]
```diff
@@ -1,5 +1,66 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-<h1>page d'inscription</h1>
+	<div class="container signUpPage">
```
Comment (benrcd) [inline src/View/Home/signup.html.twig]: Same here no camelCase for class name

File: src/View/Home/signup.html.twig [hunk 4]
```diff
@@ -1,5 +1,66 @@
...
+		<article>
+			<header>
+				<h3>Vous recevez des musiciens</h3>
+			</header>
+			<form action="" method="post">
+				<h4>Espace d'inscription hôte</h4>
+				<fieldset>
+					<label for="UserName">Votre identifiant :
+					</label>
+					<input type="text" id="UserName" name="UserName" required placeholder="Eliase">
```
Comment (benrcd) [inline src/View/Home/signup.html.twig]: Do not duplicate ids between these two forms.

**Review by benrcd — CHANGES_REQUESTED** [review]

**Review by benrcd — CHANGES_REQUESTED** [review]
Update with new develop update first

**Review by benrcd — APPROVED** [review]

**Review by victorzanotelli — COMMENTED** [review]
c'est tout bon

**Pull request merged**